downstream/modules/platform/proc-controller-creating-a-user.adoc

```diff
@@ -14,22 +14,22 @@ Normal user:: Normal users have read and write access limited to the resources (
 . From the navigation panel, select {MenuAMUsers}. 
 . Click btn:[Create user].
 . Enter the details about your new user in the fields on the *Create user* page. Fields marked with an asterisk (*) are required.
-. Normal users are the default when no *User type* is specified. To define a user as an administrator or auditor, select a *User type* checkbox.
+. Normal users are the default when no *User type* is specified. To define a user as an administrator or auditor, select a *User type* from the drop-down menu.
 +
 [NOTE]
 ====
-If you are modifying your own password, log out and log back in again for it to take effect.
+If you are modifying your own password, log out and log back in again for the change to take effect.
 ====
 +
 . Select the *Organization* to be assigned for this user. For information about creating a new organization, see xref:proc-controller-create-organization[Creating an organization].
 . Click btn:[Create user].
-
-.Next steps
-When the user is successfully created, the *User* dialog opens. From here, you can review and change the user's Teams, Roles, Tokens and other membership details.
++
+When the user is successfully created, the *User* details screen opens. From here, you can review and change the user's Teams, Roles, Tokens and other membership details.
```

Comment:
Are these UI labels and that's why they're capitalized? If yes, they should be in bold formatting.

Reply:
Suggestion: add a comma before 'and' to say, "Teams, Roles, Tokens, and other membership details."